packages/world/src/interfaces/IWorldContextConsumer.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { ERC165 } from "./ERC165.sol";
+
+// ERC-165 Interface ID (see https://eips.ethereum.org/EIPS/eip-165)
+bytes4 constant WORLD_CONTEXT_CONSUMER_INTERFACE_ID = IWorldContextConsumer._msgSender.selector ^
+  IWorldContextConsumer._msgValue.selector ^
+  IWorldContextConsumer._world.selector ^
+  ERC165.supportsInterface.selector;
+
+interface IWorldContextConsumer is ERC165 {
+  function _msgSender() external view returns (address);
+  function _msgValue() external view returns (uint256);
+  function _world() external view returns (address);
```

Comment:
why are these needed?

Reply:
need them to be public to be part of the interface to be able to distinguish a `WorldContextConsumer` contract